site/src/_pages/spec/1_block-types.mdx

```diff
@@ -204,12 +206,14 @@ Any operations specified in the protocol should be useful in a wide variety of c
 </Frame>
 
 ```block-function
-createEntities<T>(actions: CreateEntitiesAction<T>[]): Promise<T[]>
+createEntities(actions: CreateEntitiesAction[]): Promise<Entity[]>
```

Comment:
These generics were confusing some people and aren't accurate anyway, since the returned thing has more properties than the input (e.g. identifying fields)